uint16 nb_nodes
uint16[] polygons
uint16[] nodes_x
uint16[] nodes_y
uint16 nb_path_nodes
uint16[] path
float32 heading
